jobsdb/integration_test.go

```diff
@@ -495,7 +495,7 @@ func TestJobsDB(t *testing.T) {
 
 		jobDB.MaxDSRetentionPeriod = time.Second
 
-		jobs := genJobs(defaultWorkspaceID, customVal, 1, 1)
+		jobs := genJobs(defaultWorkspaceID, customVal, 10000, 1)
```

Comment:
why did we have to increase it to such a large number?

Reply:
To make sure that the migration happened due to maxdsretention check and not because of other conditions.

Reply:
not sure I can follow, can't we achieve the same with a lower number of jobs and different configuration options?

Reply:
reduced to meaningful number (10).